Automattic/jetpack
# Stats: load assets from new location `odyssey-stats`

Fixes [#71111](https://github.com/Automattic/wp-calypso/issues/71111)

#### Changes proposed in this Pull Request:

The PR proposes to load Odyssey Stats from its new location `widgets.wp.com/odyssey-stats`, and also change all the reference from `Calypso Stats` to `Odyssey Stats`.

#### Other information:


#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Set up env with Option 2 in `PejTkB-3E-p2`
* Enable new Stats experience with instruction in the P2 as well
* Apply `D94962-code` to your sandbox
* Open `/wp-admin/admin.php?page=stats`
* Ensure everything generally looks and feels okay
* Ensure links etc all work
* If anything's broken please raise bugs in [the project board](https://github.com/orgs/Automattic/projects/484)

![image](https://user-images.githubusercontent.com/1425433/207999720-2ea3b345-3e75-41bc-9ec2-44f06acd7957.png)
